manage/installing/installation_minimal_buildout.rst

```diff
@@ -0,0 +1,77 @@
+======================================
+Install Plone Using A Minimal Buildout
+======================================
+
+.. note ::
+
+   This example is adequate for a quick evaluation.
+
+   **Do not use it in production !**
+
+
+Create a directory called `Plone-5` and enter it:
+
+.. code-block:: shell
+
+   mkdir Plone-5
+   cd Plone-5
+
+Create a virtual python environment (`virtualenv <https://virtualenv.pypa.io/en/stable/>`_) and install `zc.buildout <https://pypi.python.org/pypi/zc.buildout>`_:
+
+.. code-block:: shell
+
+   virtualenv-2.7 zinstance
+   cd zinstance
+   bin/pip install zc.buildout
+
+Create a buildout.cfg file:
+
+.. code-block:: ini
+
+  echo """
```

Comment:
@ale-rt thanks ! But I 'just' copied the stuff to the new location, so it is not me who deserves the thank you, it is the awesome person who wrote these docs in the first place ! :)